packages/cli/src/commands/devHandler.js

```diff
@@ -15,6 +16,35 @@ import { getFreePort } from '../lib/ports'
 
 const defaultApiDebugPort = 18911
 
+// @TODO Remove as of Redwood 7.0.0
+const jsDeprecationNotice = () => {
+  if (process.env.REDWOOD_DISABLE_JS_DEPRECATION_NOTICE) {
+    return
+  }
+
+  // There may be actual legitimate JS-only files on the web side, so don't
+  // search ALL files, just the main ones like App, Routes, pages and components
+  const matches = fg.sync(
+    [
+      'web/src/App.js',
+      'web/src/Routes.js',
+      'web/src/components/**/*.js',
+      'web/src/pages/**/*.js',
+    ],
+    { ignore: ['web/src/**/*.mock.js', 'web/src/**/*.routeHooks.js'] }
```

Comment:
Think we need to ignore tests, .spec, anything that starts with `.` e.g. config files `.eslintrc.js`

Reply:
I'm not sure if it actually makes any difference, but is it worth setting the `cwd` to web/src instead of the root of the package? Feels like it would have to do less filtering - just a thought. 

Looking at the ignore block here https://github.com/redwoodjs/redwood/blob/main/packages/internal/src/files.ts#L68-L73

Maybe we could do:
```
      '**/*.test.{js}',
      '**/*.fixtures.{js}',
      '**/*.mock.js'
```

Reply:
Added!